docs/pages/components/advanced-link-form/doc.md

```diff
@@ -19,37 +19,43 @@ Inside a card:
 Inside a modal (the modal's body is set to have no body):
 
 :::sample-card
-<div class="p-4 text-center">
-  <button class="btn btn-info font-weight-bold" @click="$refs.formModal.show()">
-    Click to see the form
-  </button>
+<div>
```

Comment:
Do you want to add a surrounding template tag like in other example?

Reply:
I couldn't understand why, but sometimes it's needed, and sometime it's not 🤷‍♂️